rskj-core/src/test/java/co/rsk/peg/bitcoin/BitcoinUtilsTest.java

```diff
@@ -303,35 +303,36 @@ void test_extractRedeemScriptFromInput_migration_multiple_inputs() {
     }
 
     @Test
-    void removeSignaturesFromTransaction_whenTransactionDoesNotHaveInputs_shouldReturnSameTransaction() {
+    void getTransactionHashWithoutSignatures_whenTransactionDoesNotHaveInputs_shouldReturnExpectedTxHash() {
```

Comment:
```suggestion
    void getTransactionHashWithoutSignatures_whenTransactionDoesNotHaveInputs_shouldReturnSameTxHash() {
```

Reply:
Same for this one. If there are no inputs, the transaction hash is the same. In that case, the "expected tx" is just the transaction hash. In that case, there is no "without signature tx hash".

Reply:
updated